game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleState.java

```diff
@@ -30,10 +30,24 @@ public Side getOpposite() {
     }
   }
 
+  enum UnitsStatus {
+    // units that have not been hit
+    ALIVE,
+    // both ALIVE and CASUALTY
+    ACTIVE,
```

Comment:
I'm a bit confused by 'active' being alive and casualty. The confusing part is if they are alive and casualty, then wouldn't they be 'casualty'? 

The enum maybe would be more descriptive as well called `UnitBattleStatus` as it's seemingly the units status relative to the current battle. 

The 'active' I think could use more clarity